src/Chapter3.hs

```diff
@@ -589,7 +700,7 @@ hitPlayer player1 player2 =
         newHealth = calculatePlayerHit
             damage
             defense
-            (playerHealth player1)
+            (Health (playerHealth player1))
```

Comment:
If you return `Health` instead of `Int` directly in the `calculatePlayerHit` function, you don't need to wrap it here, and this function won't require any change at all!